docs/source/simulated_populations/index.rst

```diff
@@ -65,6 +65,9 @@ The following table provides the SHA-256 checksum for the larger-scale simulated
   * - Location
     - File
     - SHA-256 checksum
+  * - Rhode Island
```

Comment:
These may be confusing for users if there is no data for US with v_2_0_1 but would require we also add US data.

Reply:
We might consider removing the RI 2_0_0 data, since we won't offer that for download anymore, but I think it's fine as is.